runtime/lua/vim/diagnostic.lua

```diff
@@ -618,6 +594,14 @@ end
 ---                * priority: (number, default 10) Base priority to use for signs. When
 ---                {severity_sort} is used, the priority of a sign is adjusted based on
 ---                its severity. Otherwise, all signs use the same priority.
+---                * text: (table) A table mapping |diagnostic-severity| to the sign text
+---                to display in the sign column. The default is to use "E", "W", "I", and "H"
+---                for errors, warnings, information, and hints, respectively. Example:
+---                   <pre>lua
+---                     vim.diagnostic.config({
+---                       sign = { text = { [vim.diagnostic.severity.ERROR] = 'E', ... } }
```

Comment:
I think this should be `signs`, not `sign`.

Reply:
@mfussenegger I'll add this to https://github.com/neovim/neovim/pull/26555 as well.